modules/apps/segments/segments-web/src/test/java/com/liferay/segments/web/internal/odata/ExpressionVisitorImplTest.java

```diff
@@ -64,6 +73,49 @@ public void setUp() {
 		_expressionVisitorImpl = new ExpressionVisitorImpl(0, _entityModel);
 	}
 
+	@Test
+	public void testVisitBinaryExpressionOperationSub()
+		throws ExpressionVisitException, ParseException {
+
+		Duration duration1Day = Duration.ofDays(1);
```

Comment:
Since you don'¡t have more duration, following Liferay code style, the name of the variable should be duration.

Reply:
ok right, I change it. 

Thanks :)